ansible/roles/erwin-idm/templates/init_job.yml.j2

```
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: erwinidm-init-job
+  labels:
+    app: api
+spec:
+  template:
```

maybe add an label to the pod for the job
```suggestion
  template:
    metadata:
      labels:
        app: erwinidm-init
```

---

Done